src/Tribe/Venue.php

```diff
@@ -672,4 +673,15 @@ protected function get_duplicate_custom_fields() {
 		 */
 		return apply_filters( 'tribe_event_venue_duplicate_custom_fields', $fields );
 	}
+
+	/**
+	 * Add JSON LD script for the single venue page.
+	 *
+	 * @since TBD
+	 */
+	public function add_json_ld_to_single_venues() {
```

Comment:
Since we don't have single pages for Venues in The Events Calendar, we should move this to Pro.